crates/plugin_resolve/src/lib.rs

```diff
@@ -83,7 +83,7 @@ impl Plugin for FarmPluginResolve {
       // check external first, if the source is set as external, return it immediately
       if external_config.is_external(source) {
         return Ok(Some(PluginResolveHookResult {
-          resolved_path: param.source.clone(),
+          resolved_path: String::from(source),
```

Comment:
you should remove remove querystring `?xxx` from `param.source` to resolve this issue

Reply:
The querystring is removed at line 63:
```rust
    // split query from source
    let splits: Vec<&str> = source.split('?').collect();
    let source = splits[0];
```

But the `param.source` remains untouched.
So I have changed `param.source` into the `source`, which querystring is removed.